docs/10_dl_cnn_files/figure-html/unnamed-chunk-40-1.svg

```diff
@@ -25,25 +25,24 @@
-<polyline points='137.29,223.68 137.29,64.65 ' style='stroke-width: 0.53; stroke: #EBEBEB; stroke-linecap: butt;' />
-<polyline points='179.07,223.68 179.07,64.65 ' style='stroke-width: 0.53; stroke: #EBEBEB; stroke-linecap: butt;' />
-<polyline points='220.84,223.68 220.84,64.65 ' style='stroke-width: 0.53; stroke: #EBEBEB; stroke-linecap: butt;' />
-<polyline points='116.40,223.68 116.40,64.65 ' style='stroke-width: 1.07; stroke: #EBEBEB; stroke-linecap: butt;' />
-<polyline points='158.18,223.68 158.18,64.65 ' style='stroke-width: 1.07; stroke: #EBEBEB; stroke-linecap: butt;' />
-<polyline points='199.96,223.68 199.96,64.65 ' style='stroke-width: 1.07; stroke: #EBEBEB; stroke-linecap: butt;' />
-<polyline points='241.73,223.68 241.73,64.65 ' style='stroke-width: 1.07; stroke: #EBEBEB; stroke-linecap: butt;' />
-<rect x='146.34' y='166.18' width='11.84' height='12.78' style='stroke-width: 1.07; stroke: none; stroke-linecap: square; stroke-linejoin: miter; fill: #B22222; fill-opacity: 0.90;' />
-<rect x='158.18' y='208.77' width='5.73' height='12.78' style='stroke-width: 1.07; stroke: none; stroke-linecap: square; stroke-linejoin: miter; fill: #4682B4; fill-opacity: 0.90;' />
-<rect x='158.18' y='66.78' width='37.94' height='12.78' style='stroke-width: 1.07; stroke: none; stroke-linecap: square; stroke-linejoin: miter; fill: #4682B4; fill-opacity: 0.90;' />
-<rect x='158.18' y='151.98' width='13.17' height='12.78' style='stroke-width: 1.07; stroke: none; stroke-linecap: square; stroke-linejoin: miter; fill: #4682B4; fill-opacity: 0.90;' />
-<rect x='158.18' y='80.98' width='26.88' height='12.78' style='stroke-width: 1.07; stroke: none; stroke-linecap: square; stroke-linejoin: miter; fill: #4682B4; fill-opacity: 0.90;' />
-<rect x='142.45' y='123.58' width='15.73' height='12.78' style='stroke-width: 1.07; stroke: none; stroke-linecap: square; stroke-linejoin: miter; fill: #B22222; fill-opacity: 0.90;' />
-<rect x='158.18' y='194.57' width='9.99' height='12.78' style='stroke-width: 1.07; stroke: none; stroke-linecap: square; stroke-linejoin: miter; fill: #4682B4; fill-opacity: 0.90;' />
-<rect x='158.18' y='109.38' width='15.88' height='12.78' style='stroke-width: 1.07; stroke: none; stroke-linecap: square; stroke-linejoin: miter; fill: #4682B4; fill-opacity: 0.90;' />
-<rect x='158.18' y='137.78' width='13.33' height='12.78' style='stroke-width: 1.07; stroke: none; stroke-linecap: square; stroke-linejoin: miter; fill: #4682B4; fill-opacity: 0.90;' />
-<rect x='158.18' y='95.18' width='19.29' height='12.78' style='stroke-width: 1.07; stroke: none; stroke-linecap: square; stroke-linejoin: miter; fill: #4682B4; fill-opacity: 0.90;' />
-<rect x='147.82' y='180.38' width='10.36' height='12.78' style='stroke-width: 1.07; stroke: none; stroke-linecap: square; stroke-linejoin: miter; fill: #B22222; fill-opacity: 0.90;' />
+<polyline points='96.85,223.68 96.85,64.65 ' style='stroke-width: 0.53; stroke: #EBEBEB; stroke-linecap: butt;' />
```

Comment:
This is the first, wrong-colored version that we do not want to include.

Reply:
Deleted!